opensrp-opd/src/test/java/org/smartregister/opd/utils/OpdLookUpUtilsTest.java

```diff
@@ -0,0 +1,119 @@
+package org.smartregister.opd.utils;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.reflect.Whitebox;
+import org.smartregister.Context;
+import org.smartregister.commonregistry.CommonPersonObject;
+import org.smartregister.opd.BuildConfig;
+import org.smartregister.opd.OpdLibrary;
+import org.smartregister.opd.configuration.OpdConfiguration;
+import org.smartregister.opd.pojos.OpdMetadata;
+import org.smartregister.repository.Repository;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({OpdLookUpUtils.class})
+public class OpdLookUpUtilsTest {
+
+    @Rule
+    ExpectedException expectedException = ExpectedException.none();
+
+    @Test
+    public void testLookUpQueryWhenEntityMapIsNull() throws Exception {
```

Comment:
Does this test actually test a functionality that we would not want to break? Or make sure works a certain way.